astro/src/content/docs/lifecycle/authenticate-users/oauth/endpoints.mdx

```diff
@@ -1,6 +1,6 @@
 ---
-title: Endpoints
-description: A description of each of the OAuth and OIDC endpoints provided by FusionAuth.
+title: OAuth2 Endpoints | FusionAuth
```

Comment:
```suggestion
title: OAuth2 Endpoints
```

Again, looks weird :) . If you want the `| FusionAuth` in the title HTML tag for every docs page for SEO purposes, we might be able to do that, but doing it in the `title` section of the doc, which is used as the h1 header, in the nav, and for the title HTML tag looks weird.